crates/sui-indexer/src/handlers/pruner.rs

```diff
@@ -12,44 +12,48 @@ use crate::store::pg_partition_manager::PgPartitionManager;
 use crate::store::PgIndexerStore;
 use crate::{metrics::IndexerMetrics, store::IndexerStore, types::IndexerResult};
 
+lazy_static! {
+    static ref DEFAULT_EPOCHS_OVERRIDES: HashMap<&'static str, u64> = {
+        let mut m = HashMap::new();
+        m.insert("objects_history", 2);
+        m
+    };
+}
+
```

Comment:
use LazyCell instead of lazy_static

Reply:
Could we avoid the global and instead pass this config info through?

Reply:
We do that too in this PR. Here we provide a few defaults overrides.
I think if we could change our config strategy to a file instead of command line args, it would be a lot easier to move this fully to configs.